Epsilon.Host.Frontend/src/components/CompetenceProfileCell.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <td
+        class="competence-profile-data"
+        :style="{
+            backgroundColor: cellColor(props.kpis)?.color,
+        }"
+        :class="{
+            'cell-changed': hasChanged,
+        }">
+        <div class="cell-value">{{ props.kpis.length }}</div>
+    </td>
+</template>
+
+<script lang="ts" setup>
+import { watch, ref } from "vue"
+import { MasteryLevel, ProfessionalTaskResult } from "../logic/Api"
+
+const hasChanged = ref(false)
+
+const props = defineProps<{
+    name: string
+    kpis: ProfessionalTaskResult[]
+    levels: MasteryLevel[]
+}>()
+
+watch(
+    () => props.kpis,
+    (newKpis, oldKpis) => {
```

Comment:
Please make a separated function for this. Not calling it, the function directly. 
Also use "watchEffect" instead of "watch"
https://www.vuemastery.com/blog/vues-watch-vs-watcheffect-which-should-i-use/

Reply:
"watch" takes the old KPIs into account, which I need.
Discussed with Jelle that the function is unnecessary

Reply:
Agreed